transhumandesign/kag-base
# Save the Princess Improvement

## Status

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

Solving #1685 and #1687
Better mounted bow icon in list of shop, and no fanfare on checkpoint taking

## Steps to Test or Reproduce

#1685 Play Save the Princess and check shop
#1687 Play it and take the checkpoint
